core/src/Kokkos_View.hpp

```diff
@@ -123,6 +123,32 @@ void runtime_check_rank(const size_t rank, const size_t dyn_rank,
   }
 }
 
+template <class T, int R,
+          bool P = (R > 1) && (std::is_same<T, Kokkos::LayoutLeft>::value ||
+                               std::is_same<T, Kokkos::LayoutRight>::value)>
```

Comment:
```suggestion
          bool P = (R > 1) && (std::is_same_v<T, Kokkos::LayoutLeft> ||
                               std::is_same_v<T, Kokkos::LayoutRight>)>
```
Since I don't think we intend to backport this to 3.7.x, we can use the nicer C++17 spelling.

Reply:
Thanks, yep